src/ui/public/directives/pretty_duration.js

```diff
@@ -1,20 +1,20 @@
 import _ from 'lodash';
 import dateMath from '@elastic/datemath';
 import moment from 'moment';
-import 'ui/timepicker/quick_ranges';
 import 'ui/timepicker/time_units';
 import { uiModules } from 'ui/modules';
 const module = uiModules.get('kibana');
 
 
-module.directive('prettyDuration', function (config, quickRanges, timeUnits) {
+module.directive('prettyDuration', function (config, timeUnits) {
   return {
     restrict: 'E',
     scope: {
       from: '=',
       to: '='
     },
     link: function ($scope, $elem) {
+      const quickRanges = config.get('timepicker:quickRanges');
```

Comment:
Do we need a check here to make sure users don't add a crazy number of sections?

Reply:
Meh, we could, but I don't envision that being a super common problem and it's easily fixed.